client-v2/src/services/__tests__/capiQuery.spec.ts

```diff
@@ -18,7 +18,8 @@ describe('CAPI', () => {
   describe('search', () => {
     it('makes a network request on a query', () => {
       const apiKey = 'my-api-key';
-      const capi = capiQuery();
+      const capiUrl = 'https://capiurl.guardian.com';
+      const capi = capiQuery(capiUrl);
```

Comment:
I did try to extract these into a global variable, using Jest's [Globals Object ](https://jestjs.io/docs/en/configuration#globals-object). But the code came out more confusing and not much shorter. Ie: 
```
    const apiKey = (global as any).apiKey;
    const capi = capiQuery((global as any).capiUrl);
```
So opted for this longer but more readable option